cmd/skopeo/flag.go

```diff
@@ -18,7 +18,14 @@ type optionalBool struct {
 // newOptionalBool
 type optionalBoolValue optionalBool
 
-func newOptionalBoolValue(p *optionalBool) cli.Generic {
+func optionalBoolFlag(fs *pflag.FlagSet, p *optionalBool, name, usage string) {
+	if ret := fs.Lookup(name); ret == nil {
```

Comment:
Why the `Lookup` here? The Cobra implementations don’t do that, and AFAICS tests pass fine without it.